src/bin/gpfusion/gpbridgeapi.c

```diff
@@ -273,6 +277,26 @@ void gpbridge_export_start(PG_FUNCTION_ARGS)
 
 }
 
+void parse_namespace(char *uri, char** extracted_namespace){
+	int i = 0;
+	int len = strlen(uri);
+
+	while(uri[i] != '*' && i < len){
```

Comment:
Good point on the delimiter, I'll replace it.

I chose not to parse the URI in `parseGPHDUri` because even with this fix we will support two different flows for the readable and writable case. So this is a temporary solution whose effects I wanted to contain only to writable cases.